packages/frontend/.env.template.production

```diff
@@ -0,0 +1,8 @@
+
+## FRONTEND ##
+
+# Full URL to host where API is running
+REACT_APP_SERVER_URL=https://localhost
+
+# Port number used when running frontend for development, outside of Docker
+PORT=3000
```

Comment:
Do you think we need both the `.env.template.production` and `.env.template.development` files? Would just one template suffice?